org.nelson.Nelson.yaml

```
@@ -0,0 +1,314 @@
+        sha256: c261a4db579b289a7501565497658bbd52d3138fdbaccf1490fa918129ab45bc
+
+  - name: libgit2
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_SHARED_LIBS=ON
+      - -DCMAKE_C_FLAGS="-O2"
+    sources:
+      - type: archive
+        url: https://github.com/libgit2/libgit2/archive/refs/tags/v1.9.0.tar.gz
+        sha256: 75b27d4d6df44bd34e2f70663cfd998f5ec41e680e1e593238bbe517a84c7ed2
+
+  - name: libsndfile
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_SHARED_LIBS=ON
+      - -DBUILD_TESTING=OFF
+    sources:
+      - type: archive
+        url: https://github.com/libsndfile/libsndfile/archive/refs/tags/1.2.2.tar.gz
+        sha256: ffe12ef8add3eaca876f04087734e6e8e029350082f3251f565fa9da55b52121
+
+  - name: taglib
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://github.com/taglib/taglib/releases/download/v2.0.1/taglib-2.0.1.tar.gz
+        sha256: 08c0a27b96aa5c4e23060fe0b6f93102ee9091a9385257b9d0ddcf467de0d925
+
+  - name: giflib
+    buildsystem: simple
+    build-commands:
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make PREFIX=/app install
+    sources:
+      - type: archive
+        url: https://downloads.sourceforge.net/project/giflib/giflib-5.2.1.tar.gz
+        sha256: 31da5562f44c5f15d63340a09a4fd62b48c45620cd302f77a6d9acf0077879bd
+
+  - name: libtiff
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_SHARED_LIBS=ON
+    sources:
+      - type: archive
+        url: https://download.osgeo.org/libtiff/tiff-4.6.0.tar.gz
+        sha256: 88b3979e6d5c7e32b50d7ec72fb15af724f6ab2cbf7e10880c360a77e4b5d99a
+
+  - name: openblas
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_SHARED_LIBS=ON
+      - -DUSE_OPENMP=ON
+    sources:
+      - type: archive
+        url: https://github.com/xianyi/OpenBLAS/archive/v0.3.28.tar.gz
+        sha256: f1003466ad074e9b0c8d421a204121100b0751c96fc6fcf3d1456bd12f8a00a1
+
+  - name: lapack
+    buildsystem: cmake
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DBUILD_SHARED_LIBS=ON
+      - -DLAPACKE=ON
+      - -DCBLAS=ON
+    sources:
+      - type: archive
+        url: https://github.com/Reference-LAPACK/lapack/archive/v3.12.0.tar.gz
+        sha256: eac9570f8e0ad6f30ce4b963f4f033f0f643e7c3912fc9ee6cd99120675ad48b
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=/app --with-libraries=all
+      - ./b2 headers
+      - ./b2 install --prefix=/app variant=release threading=multi link=shared runtime-link=shared -j$(nproc)
```

```suggestion
      - ./b2 install --prefix=/app variant=release threading=multi link=shared runtime-link=shared -j$FLATPAK_BUILDER_N_JOBS
```

never use nproc for that.